### PR TITLE
Focus

### PR DESCRIPTION
**Pull request by Husain01**
-> User can add main focus for today
-> User can edit the focus
-> User can check the focus
-> User can remove and add another focus.

### CODE REVIEW

**Review by Jia-A — APPROVED** [review]
Your code is really well structured and clean. Also there is good use of comments.
You can merge it.

**Review by OmKakatkar — COMMENTED** [review]
image
Hi,
The form is not getting submitted on pressing enter and there doesn't seem to be any submit button for the same. Is this feature still under development

File: src/pages/Welcome/Welcome.css [hunk 1]
```diff
@@ -1,93 +1,93 @@
...
-    border-radius: 10px;
-    border: 1px solid rgba( 255, 255, 255, 0.18 );
+.input {
+  outline: none;
+  padding: 0.5rem 1rem;
+  font-size: 1.5rem;
+  text-align: center;
+  color: white;
+  margin-top: 1rem;
+  background: rgba(255, 255, 255, 0.25);
```
Comment (OmKakatkar) [inline src/pages/Welcome/Welcome.css:21]: Suggestion : The colors are being used at multiple places you can use a variable for the same

**Review by OmKakatkar — APPROVED** [review]
Like : The UI looks clean. The features are working as expected.
Suggestion : Delete old site previews from the comments.

File: src/App.css [hunk 2]
```diff
@@ -1,28 +1,25 @@
-.App{
-/* z-index: -1; */
-height: 100vh;
-background-repeat: no-repeat;
+.App {
+  /* z-index: -1; */
```
Comment (manojasarsa) [inline src/App.css:2]: You can remove comments if no longer required.

**Review by manojasarsa — APPROVED** [review]
The code seems fine. Good to merge.

**Pull request merged**